### PR TITLE
Add issue #156 workaround to README.md

### PR DESCRIPTION
**Pull request by henrik-ch**
Suggestion to add the #156 workaround to README.md.

### CODE REVIEW

**Review by nzbr — APPROVED** [review]
I didn't know the `<details>` block even existed. That's neat!

File: README.md [hunk 1]
```diff
@@ -23,6 +23,14 @@ This sets up a new WSL distribution `NixOS` that is installed under
 `.\NixOS`. `nixos-wsl-installer.tar.gz` is the path to the file you
 downloaded earlier. You might need to change this path or change to the download directory first.
 
+<details>
+    <summary>Did Install hang with "Starting Systemd.." or do you get error message `nsenter: failed to parse pid: '-p'`?</summary>
```
Comment (SuperSandro2000) [inline README.md:27]: Using markdown code blocks does not work inside of html code

Comment (henrik-ch) [reply README.md]: Thanks for feedback @SuperSandro2000 , should be better now I hope.

File: README.md [hunk 2]
```diff
@@ -23,6 +23,14 @@ This sets up a new WSL distribution `NixOS` that is installed under
 `.\NixOS`. `nixos-wsl-installer.tar.gz` is the path to the file you
 downloaded earlier. You might need to change this path or change to the download directory first.
 
+<details>
+    <summary>Did Install hang with <code>Starting Systemd..</code> or do you get error message <code>nsenter: failed to parse pid: '-p'</code>?</summary>
+
+## Workaround
```
Comment (SuperSandro2000) [inline README.md:29]:
```suggestion
### Workaround
```

**Pull request closed without merging**